README.md

```diff
@@ -582,6 +582,28 @@ A LightOperation is an object containing at least one of a `Light`'s properties,
 ```
 or a subset thereof.
 
+### `Plug`
+A plug represents a single outlet plug and has several properties describing its state:
+* `onOff: boolean` - If the plug is on (`true`) or off (`false`)
+
+as well as a few readonly properties:
+* `isSwitchable: boolean` - Whether the plugs supports on/off (always true).
+* `isDimmable: boolean` - Whether the plug supports setting the dimmer value (always false for now).
+
+If the plug object was returned from a library function and not created by you, the following methods are available to change its appearance directly. You can await them to make sure the commands were sent or just fire-and-forget them. The returned Promises resolve to true if a command was sent, otherwise to false.
+* `turnOn()` - Turns the plug on.
+* `turnOff()` - Turns the plug off.
+* `toggle([value: boolean])` - Toggles the plug's state to the given value or the opposite of its current state.
+
+### `PlugOperation`
+A PlugOperation is an object containing at least one of a `Plug`'s properties, which are:
+```TS
+{
+    onOff: boolean;
+}
+```
+or a subset thereof.
```

Comment:
A subset of 
```ts
{
    onOff: boolean;
}
```
is an empty object which won't be transmitted. This would only make sense if there's more than 1 property (like with lights).